zebra-chain/src/sapling/commitment.rs

```diff
@@ -234,4 +286,102 @@ mod tests {
             assert_eq!(result, test_vector.output_point);
         }
     }
+
+    #[test]
+    fn test_add() {
```

Comment:
If we ever get a good way to generate arbitrary value commitments this would make a good proptest

Reply:
Yeah making sure 0 + <random> = <random> seems fine for a proptest, too bad we don't have a cheap strategy for these points yet